sum.test.js

```diff
@@ -0,0 +1,5 @@
+const sum = require('./sum');

```

Comment:
This looks like something from development that should probably not be in the main codebase.